ocs_ci/ocs/constants.py

```diff
@@ -2052,3 +2052,54 @@
 
 
 WARP_CLIENT_PORT = 7761
+
+UI_INPUT_RULES_GENERAL = {
+    "rule1": "Starts and ends with a lowercase letter or number",
+    "rule2": "Only lowercase letters, numbers, non-consecutive periods, or hyphens",
+    "rule3": "A unique name for the BackingStore within the project",
+    "rule4": "Cannot be used before",
+    "rule5": "No more than 253 characters",
+}
+
+UI_INPUT_RULES_BACKING_STORE = {
+    "rule1": "No more than 43 characters",
+    "rule2": UI_INPUT_RULES_GENERAL["rule1"],
```

Comment:
Do we really need to rename rule1 as rule2, rule5 as rule1 and so on? Wouldn't it be nicer to have some consistency?

Reply:
all the keys are sorted by numbers "rule1", "rule2", "rule3" for every specific list of rules but not general list of rule.
UI_INPUT_RULES_GENERAL are general rules repeated from one popup to another, thus we use them when required and do not repeat ourself.
we call UI_INPUT_RULES_GENERAL when need to take a rule from common list. This list can not be consistent both for general rules UI_INPUT_RULES_GENERAL and for all the specific case rules such as UI_INPUT_RULES_BUCKET_CLASS, UI_INPUT_RULES_OBJECT_BUCKET_CLAIM or UI_INPUT_RULES_STORAGE_SYSTEM
---------
calling specific rule, such as UI_INPUT_RULES_BUCKET_CLASS or UI_INPUT_RULES_NAMESPACE_STORE or any else we rely on consequent num "rule1", "rule2", "rule3", etc and do not see underlying relation with UI_INPUT_RULES_GENERAL. Basically UI_INPUT_RULES_GENERAL should not be used outside of the constants file